test/contracts/OVM/chain/OVM_CanonicalTransactionChain.spec.ts

```diff
@@ -500,7 +508,7 @@ describe('OVM_CanonicalTransactionChain', () => {
   })
 
   describe('verifyTransaction', () => {
-    it('should successfully verify against a valid queue transaction', async () => {
+    it.skip('should successfully verify against a valid queue transaction', async () => {
```

Comment:
I actually think it's important to keep this test since the code path being tested has not been disabled, just the way we were getting there.  Recommend to replace line 523 with a `sequencerBatchAppend` and un-skipping.